examples/sub-class.py

```diff
@@ -29,36 +29,30 @@
         sys.path.insert(0, cmd_subfolder)
     import paho.mqtt.client as mqtt
 
-class MyMQTTClass:
-    def __init__(self, clientid=None):
-        self._mqttc = mqtt.Client(clientid)
-        self._mqttc.on_message = self.mqtt_on_message
-        self._mqttc.on_connect = self.mqtt_on_connect
-        self._mqttc.on_publish = self.mqtt_on_publish
-        self._mqttc.on_subscribe = self.mqtt_on_subscribe
+class MyMQTTClass(mqtt.Client):
 
-    def mqtt_on_connect(self, mqttc, obj, flags, rc):
+    def on_connect(self, mqttc, obj, flags, rc):
```

Comment:
To clear up a point of confusion, to give "sub-class.py" it's full name you'd write "an-example-of-a-subscribing-client-that-uses-a-class.py", not "an-example-of-a-client-using-subclassing.py". I think I'll rename it to be a bit clearer.

It's also clear that using an MQTT client object rather than inheriting is the most common pattern in use.